features/step_definitions/filtering_steps.rb

```diff
@@ -60,6 +60,16 @@
   expect(page).to have_css(".finder-results .gem-c-document-list__item", count: 1)
 end
 
+Then(/I should see the keyword$/) do
+  expect(page).to have_content(@keyword_search)
```

Comment:
Ah yes, sorry that was left over from an earlier approach. Thanks for catching. 

Seems low value to test the search box in this respect since we've just entered text into it.